tests/lib/assertions/valid-for-testing-pc-revno-2-20.json

```diff
@@ -0,0 +1,43 @@
+{
+    "type": "model",
+    "authority-id": "FpA2tgNE8bUrUKSOhE6SOpsMdlcEJkDL",
+    "series": "16",
+    "brand-id": "FpA2tgNE8bUrUKSOhE6SOpsMdlcEJkDL",
+    "model": "my-model",
+    "architecture": "amd64",
+    "timestamp": "2021-06-01T12:18:12+00:00",
+    "grade": "dangerous",
+    "base": "core20",
+    "serial-authority": ["generic"],
+    "snaps": [
+        {
+            "default-channel": "20/edge",
+            "id": "UqFziVZDHLSyO3TqSWgNBoAdHbLI4dAH",
+            "name": "pc",
+            "type": "gadget"
+        },
+        {
+            "default-channel": "20/edge",
+            "id": "pYVQrBcKmBa0mZ4CCN7ExT6jH8rY1hza",
+            "name": "pc-kernel",
+            "type": "kernel"
+        },
+        {
+            "default-channel": "latest/stable",
+            "id": "DLqre5XGLbDqg9jPtiAhRRjDuPVa5X1q",
+            "name": "core20",
+            "type": "base"
+        },
+        {
+            "default-channel": "latest/stable",
+            "id": "PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4",
+            "name": "snapd",
+            "type": "snapd"
+        },
+        {
+            "default-channel": "latest/stable",
+            "name": "test-snapd-tools-core20"
+        }
+    ],
+    "revision": "2"
```

Comment:
this would be easier to notice if it were up top with `snaps` the last key in the object

Reply:
I moved the revision tot he very top so that it stands out a bit more